packages/jaeger-ui/src/components/common/ErrorMessage.test.js

```diff
@@ -13,141 +13,124 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow, mount } from 'enzyme';
-
+import { screen, render } from '@testing-library/react';
 import ErrorMessage, { Details } from './ErrorMessage';
 
-describe('<ErrorMessage>', () => {
-  let wrapper;
-  let error;
+import '@testing-library/jest-dom';
 
-  beforeEach(() => {
-    error = 'some-error';
-    wrapper = shallow(<ErrorMessage error={error} />);
-  });
+describe('<ErrorMessage>', () => {
+  const errorMessage = 'some error message';
 
   it('is ok when not passed an error', () => {
-    wrapper.setProps({ error: null });
-    expect(wrapper).toBeDefined();
+    render(<ErrorMessage />);
+    expect(screen).toBeDefined();
   });
 
   it('renders a message when passed a string', () => {
-    const msg = wrapper.find('Message');
-    expect(msg.length).toBe(1);
-    expect(msg.shallow().text()).toMatch(error);
+    render(<ErrorMessage error={errorMessage} />);
+    expect(screen.getByText(errorMessage)).toBeInTheDocument();
   });
 
   it('<Details /> renders empty on string error', () => {
-    wrapper = shallow(<Details error={error} />);
-    expect(wrapper).toMatchSnapshot();
+    render(<Details error={errorMessage} />);
+    expect(screen).toBeDefined();
   });
 
   it('<Details /> renders wrapper on wrap', () => {
-    error = {
-      message: 'some-http-ish-message',
+    const error = {
```

Comment:
this struct is repeated many times, maybe pull into a global const?

Reply:
Tried doing that it causes more redirection and hard-to-read tests than it reduces duplication. The primary reason for the same being each test changes/deletes just one key of the object. 
If we were to do that, then in each test we would first need to restore the global const from previous test, and then augment it again according to it's need.